internal/pkg/cli/storage_init.go

```diff
@@ -139,11 +140,19 @@ var engineTypes = []string{
 	engineTypePostgreSQL,
 }
 
+var (
+	ddbFlags                       = []string{storagePartitionKeyFlag, storageSortKeyFlag, storageNoSortFlag, storageLSIConfigFlag, storageNoLSIFlag}
+	ddbFlagExclusiveWithAddIngress = ddbFlags
```

Comment:
Can we use cobra's [MarkFlagsMutuallyExclusive](https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive) method and get rid of the internal bool flag `configFlagExclusiveWithAddIngress`?

Reply:
done!